hieradata_aws/integration.yaml

```diff
@@ -40,6 +40,7 @@ govuk::apps::email_alert_api::delivery_request_threshold: "3000"
 govuk::apps::email_alert_frontend::subscription_management_enabled: true
 govuk::apps::feedback::govuk_notify_reply_to_id: 'fee22233-2f28-4b0b-8b6c-4410979f2275'
 govuk::apps::feedback::govuk_notify_template_id: 'eb9ba220-7d74-4aab-975a-bdbe718f69a3'
+govuk::apps::finder_frontend::feature_flag_accounts: 'disabled'
```

Comment:
Can this be a boolean, rather than a string which is either `enabled` or `disabled`?

Reply:
Could do, but then would need to be changed here and across all the docker images too.

https://github.com/alphagov/finder-frontend/blob/master/config/application.rb#L65

At the time the thinking was that this was more descriptive, and would also make for less ambiguity about a pulling in a truthy value from ENV?

Reply:
No, I meant can this hieradata field be a bool, which sets the env var to `feature_flag_accounts ? "enabled" : undef`

Reply:
Environment variables contain strings, there's no getting around that

Reply:
Done